lib/sober_swag/output_object/view.rb

```diff
@@ -21,26 +28,40 @@ def initialize(name, base_fields = [])
 
       attr_reader :name, :fields
 
+      ##
+      # Serialize an object according to this view.
       def serialize(obj, opts = {})
         serializer.serialize(obj, opts)
       end
 
+      ##
+      # Get the type of this view.
       def type
         serializer.type
       end
 
+      ##
+      # Excludes a field with the given name from this view.
+      # @param name [Symbol] field to exclude.
       def except!(name)
         @fields.reject! { |f| f.name == name }
       end
 
+      ##
+      # Always raises {NestingError}
+      # @raise {NestingError} always
       def view(*)
         raise NestingError, 'no views in views'
       end
 
+      ##
+      # Adds a field do this view.
       def add_field!(field)
         @fields << field
       end
 
```

Comment:
Replace line 71,
https://github.com/SonderMindOrg/sober_swag/blob/better-rdoc-documentation/lib/sober_swag/output_object/view.rb#L71

```suggestion
      # @note WARNING Don't add more fields after you call this.
```